src/main/java/dev/notebook/app/notebook/api/controllers/Init.java

```diff
@@ -0,0 +1,51 @@
+package dev.notebook.app.notebook.api.controllers;
+
+import dev.notebook.app.notebook.entity.Note;
+import dev.notebook.app.notebook.entity.Notebook;
+import dev.notebook.app.notebook.service.NoteService;
+import dev.notebook.app.notebook.service.NotebookService;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Init {
```

Comment:
```suggestion
public class InitService {
```
let's rename to ..Service, or ...Initializer, or smth like that